tests/ssg_test_suite/template.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+from __future__ import print_function
+
+import logging
```

Comment:
Does it make sense to add some logging.info stating that it's running in template mode, otherwise this import seems unnecessary

Reply:
Looking at the logging aspect with more attention I noticed that all messages are related to rule ID, and it is confusing when using the template mode. At the same time, there is no easy way to update just the logging messages.
```
$ python3 tests/test_suite.py template  --libvirt qemu:///session rhel86 --datastream build/ssg-rhel8-ds.xml root_permissions_syslibrary_files
Setting console output to log level INFO
b^[[B;qINFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/wsato/git/content/logs/template-custom-2022-05-12-1741/test_suite.log
ERROR - No tests found matching the rule ID(s) 'root_permissions_syslibrary_files'
WARNING - Nothing has been tested!
```

I could move the strings to Rule's class definition, and when the Template class inherits from it, it can overwrite the messages. What do you think?